doc/manual/generate-store-implementation.nix

```
@@ -0,0 +1,55 @@
+let
+  inherit (builtins)
+    attrNames attrValues fromJSON listToAttrs mapAttrs
+    concatStringsSep concatMap length lessThan replaceStrings sort;
+  inherit (import ./utils.nix) optionalString filterAttrs trim squash unique showSettings;
+in
+
+commandDump:
+
+let
+
+  inherit (fromJSON commandDump) stores;
+
+  showStore = name: { settings, doc, experimentalFeature }:
+    let
+      experimentalFeatureNote = optionalString (experimentalFeature != null) ''
+        > **Warning**
+        > This store is part of an
+        > [experimental feature](@docroot@/contributing/experimental-features.md).
+
+        To use this store, you need to make sure the corresponding experimental feature,
+        [`${experimentalFeature}`](@docroot@/contributing/experimental-features.md#xp-feature-${experimentalFeature}),
+        is enabled.
+        For example, include the following in [`nix.conf`](#):
```

```suggestion
        For example, include the following in [`nix.conf`](#):
```

This should be a docroot link